src/lib/Microsoft.Health.Fhir.Ingest/Telemetry/ExceptionTelemetryProcessor.cs

```diff
@@ -31,33 +30,71 @@ public ExceptionTelemetryProcessor()
         {
         }
 
-        public ExceptionTelemetryProcessor(params Type[] handledExceptionTypes)
+        public ExceptionTelemetryProcessor(params System.Type[] handledExceptionTypes)
         {
-            _handledExceptions = new HashSet<Type>(handledExceptionTypes);
+            _handledExceptions = new HashSet<System.Type>(handledExceptionTypes);
         }
 
-        public virtual bool HandleException(Exception ex, ILogger log)
+        public virtual bool HandleException(Exception ex, ITelemetryLogger log, string connectorStage)

```

Comment:
This signature takes an ILogger because it needed to be passed into the function method.  This can probably be refactored to be injected in the ctor with DI and have a cleaner signature.  I can't see a reason currently to keep it as part of the method signature.  Our ITelemetryLogger is locked in during run time currently.

Reply:
I attempted to refactor this, but wasn't able to get it to work with DI, and need to spend some more time understanding DI. Do you mind if we do not address this in this PR?